hw1/task1/matrices.py

```diff
@@ -0,0 +1,37 @@
+from vectors import *
+import copy
+
+
+class Matrix:
+    content: List[Vector]
+
+    def __init__(self, first: Vector, *args: Vector):
+        if not all(len(first) == len(vector) for vector in args):
+            raise TypeError("Rows must be the same length")
+        self.content = [first, *args]
+
+    def __str__(self):
+        return str(self.content)
+
+    def __add__(self, other):
+        return Matrix(*[get_vector_sum(x, y) for x, y in zip(self.content, other.content)])
```

Comment:
Here, you should also check that the matrices have the same dimensions, otherwise your function can sum matrices with different sizes by only summing the "short" part.